content-src/components/Base/Base.jsx

```diff
@@ -133,6 +133,8 @@ export class BaseContent extends React.PureComponent {
   }
 
   onWindowScroll() {
+    if (!this.props.Prefs.values.showSearch) { return; }
```

Comment:
@k88hudson ☝️ like this?

Reply:
I'm testing that, I think it's ok

Reply:
Yep, that works

Reply:
Actually on second thought, this doesn't work properly if you flip search off in one state (i.e. sticky) and turn it back on in the non-sticky state. I think you should just remove it. Oh well!